dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -1850,4 +1850,59 @@ public void testCreateItemInvalidCollectionBadRequestException() throws Exceptio
                         .andExpect(status().isBadRequest());
     }
 
+    @Test
+    public void updateTestEPersonWithoutPermissionForbidden() throws Exception {
+        //We turn off the authorization system in order to create the structure as defined below
+        context.turnOffAuthorisationSystem();
```

Comment:
It appears you are never turning the AuthorizationSytem back *on*.  While it appears this isn't required for the tests to perform properly, we should *always* have a ` context.restoreAuthSystemState();` to ensure it is turned back on before performing tests

Reply:
Thank you for locating this, we'll fix this and update it shortly!